Automattic/newspack-newsletters
# Replace SVGs with PNG in social icons folder

### All Submissions:






### Changes proposed in this Pull Request:



Gmail does not like SVGs, as it turns out. 

Closes #38

### How to test the changes in this Pull Request:

1. Create a newsletter with a social icons block
2. Send the campaign or send a test email
3. Open the email in Gmail client
1. The social icons should should be visible

### Other information:
